src/ensembl/src/content/app/entity-viewer/shared/helpers/entity-helpers.ts

```diff
@@ -1,8 +1,20 @@
-import { SliceWithLocationOnly } from 'src/content/app/entity-viewer/types/slice';
+import {
+  Slice,
+  SliceWithLocationOnly
+} from 'src/content/app/entity-viewer/types/slice';
 
 export const getFeatureCoordinates = (feature: {
   slice: SliceWithLocationOnly;
 }) => {
   const { start, end } = feature.slice.location;
   return { start, end };
 };
+
+export const getFeatureStrand = (feature: { slice: Slice }) =>
+  feature.slice.region.strand;
+
+export const getFeatureLength = (feature: { slice: Slice }) => {
```

Comment:
Can you add a FIXME here so that we use feature.slice.location.length property when it becomes available and not calculate it ourselves (we will have problems with circular chromosomes if we try to do it ourselves)?

Reply:
✔️